client/src/components/Reviews/formComponents/Characteristics.jsx

```diff
@@ -16,72 +17,74 @@ const descriptions = {
     '1/2 a size too small',
     'Perfect',
     '1/2 a size too big',
-    'A size too wide'
+    'A size too wide',
   ],
   Length: [
     'Too narrow',
     'Slightly narrow',
     'Perfect',
     'Slightly wide',
-    'Too wide'
+    'Too wide',
   ],
   Comfort: [
     'Uncomfortable',
     'Slightly uncomfortable',
     'Ok',
     'Comfortable',
-    'Perfect'
+    'Perfect',
   ],
   Quality: [
     'Poor',
     'Below average',
     'What I expected',
     'Pretty great',
-    'Perfect'
-  ]
+    'Perfect',
+  ],
 };
 
 const useStyles = makeStyles(theme => ({
-  root: {
-    display: 'flex'
-  },
-  formControl: {
-    margin: theme.spacing(0)
+  category: {
+    margin: theme.spacing(0),
+    fontSize: 15,
+    fontWeight: 'bold',
   },
   group: {
-    margin: theme.spacing(2, 0)
+    margin: theme.spacing(1, 0),
+    marginBottom: theme.spacing(4),
   },
   col: {
     margin: theme.spacing(0),
-    width: 120
-  }
+    width: 80,
+    alignContent: 'center',
+  },
+  label: {
+    fontSize: 13,
+    textAlign: 'center',
+  },
 }));
 
 const Characteristics = ({ form, setForm, characteristics }) => {
-  // console.log(characteristics);
   const handleChange = e => {
     setForm(prevState => {
-      return { ...prevState, [e.target.name]: e.target.value };
+      prevState.characteristics[e.target.name] = e.target.value;
+      return { ...prevState };
```

Comment:
what this return is used for?

Reply:
It's part of react hooks, it is to return the new state so that it updates. It replaces setState